### ISSUE TITLE
Fasthax freezes on N3DS 11.2.0-35E

### ISSUE THREAD

**Issue by MasterDiablo24** [nedwill/fasthax#46]
After executing fasthax my 3DS freezes. The top screen remains black and the bottom screen changes shades of red.

note: before this happened
warning expected one reachable node, found 59
Found parent and orphan: 0xfff75258 -> 0xfff74e20
k11_exploit succeeded

**Comment by legoj15** [comment]
Happens on O3DS 11.1.0-34U, too.

**Comment by nneto** [comment]
I tried both Beta 2 and Beta 3 5+ times each on my O3DS 11.2.0-35U. Both of them ask to press start when finished and then the top screen become black and the bottom red, and remains frozen. Beta 1 works however.

edit: to me this is what is shown:
beginning attempt 0x1300
we won!
[+] UAF succeeded and local backdoor installed.
[!] Warning: expected one unreachable node, found 59!
Found parent and orphan: 0xfff74e44 -> 0xfff74a48
k11_exploit succeeded!
waiting for user... press <start> to continue

**Comment by MasterDiablo24** [comment]
I tried beta 1 several times on my N3DS 11.2.0-35E and it does not work

**Comment by nneto** [comment]
@MasterDiablo24 what happens specifically?
Despite freezing sometimes in one of the "beginning attempt [...]" (and this also happens with Beta 2 and 3) and having to restart the console, generally the fasthax has sucess (although it shows a message of failure in the end, it's possible to press start, get back to the HBL, open fasthax again and it says the backdoor is already installed...)

edit: what exactly appears is (after lots of "[+] found child: (0x0 ->)" and "[!] Warning: unexpectedly found more than one NULL prev.":
[+] found child: (0x0 ->) 0xfff7581c
[-] Failed to find parent for broken link search.
[-]Failed to find broken link in linked list.
[-] Warning! Exploit succeeded couldn't cleanup kernek.
[-] System instability may occur.
**k11_exploit failed!**
waiting for user... press <start> to continue

but when pressed start and open fasthax again:
[+] Backdoor already installed.
**k11_exploit succeeded!**
waiting for user... press <start> to continue

**Comment by MasterDiablo24** [comment]
[+] UAF succeeded and local backdoor installed.
[-] Warning: expected one unreachable node, found 59!
found parent and orphan: 0xfff75258 -> 0xfff74e20
k11_exploit succeeded!
waiting for user... press <start> to continue

However. When I press start, my top screen goes black, my bottom screen goes red and my 3ds keeps frozen like this until I hard reset.

**Comment by nneto** [comment]
@MasterDiablo24 are you sure you tried Beta 1? Your message seems more like the one I get from Beta 3. If you are having this issue with Beta 1 really, that's odd, hope ned can help you :/

**Comment by MasterDiablo24** [comment]
@NNeto This message is from beta 3, but I haven't had any luck with beta 1 either.

**Comment by MasterDiablo24** [comment]
After trying a few more times with beta 1, it finally worked. Thanks

**Comment by nneto** [comment]
@MasterDiablo24 let the issue as open, cause of the Beta 2 and 3

**Comment by pappersdrake** [comment]
Just wanted to chime in that I've tried both beta 1 and beta 3 20+ times on O3DS 11.2.0-35E but alas no attempt has succeeded, there is no text showing on the screen either, only a black top screen and red bottom screen. Hope you succeed!

**Comment by ricrui3** [comment]
I got this problem too in beta 3 in and O3DS 10.6.0-31U. There was no problem once i used the beta 1

**Comment by mijodong** [comment]
Getting similar errors with Betas 1, 2, and 3 on N3DS 11.2.0-35U

**Comment by chivs688** [comment]
I have the exact same black top screen, red bottom screen as soon as I press Start using Beta 3 on my N3DSXL 11.2.0.

Tried Beta 1 but got a looad of error messages, resulting in:

[-] Warning! Exploit succeeded couldn't cleanup kernel.
[-] System instability may occur.
k11_exploit failed!

Have not pressed Start after that in fear.

**Comment by anthus1** [comment]
I am trying this on a n3DS XL running 11.1. I have tried it a few times, and am getting the same red screen crashes others have numerously reported. I'm following the 3ds.guide page. Here is a picture of my screen. 

https://www.dropbox.com/s/qenc2t6vyrhmpk0/IMG_20170108_210652.jpg?dl=0

It says it succeeded, but it always hangs and required a hard reset after pressing start. Does this mean it actually didn't work? I don't want to continue with it unless I'm sure it worked. Also, is it normal for the HBL to not load sometimes, hanging on a white bottom screen, and a line on the top saying "the homemenu robpin is ready"?

Thanks guys, for all this stuff you have done. Looking forward to getting my fully working 3DS modded. I was able to install a9hl on an old 3ds, with tubehax, but that one is water damaged, and not really playable in some cases.
